src/main/java/io/github/syst3ms/skriptparser/effects/EffExit.java

```diff
@@ -105,13 +112,17 @@ public String toString(TriggerContext ctx, boolean debug) {
             if (parseMark == 0
                     || parseMark == 1 && (stm instanceof SecLoop || stm instanceof SecWhile)
                     || parseMark == 2 && stm instanceof Conditional) {
+                if (stm instanceof Finishing)
```

Comment:
> not about the line but more about the method, i suggest to add Javadoc because this method isn't clear imo (or not easly understandable)

I added Javadocs (see the Finishing class for the documentation. Can you read it and perhaps see if it is clear enough?

Reply:
I mean add javadoc for escapeSection method